### DISCUSSION TITLE
Max video duration after labs update

### DISCUSSION THREAD

**Discussion by i-marinkovic** [gopro/labs#1041] [Q&A]
Hi friends,
I need to ask if someone noted after last lab firmware update, the duration of a 360 video file is about 14 minutes, instead previous 8 minutes per file.

What changed?

**Comment by dnewman-gpsw** [comment]
The default used to be 4GB chapters at 60Mb/s.  4000*8/60 = 534 seconds ~ 8.9 minutes.
It now seems to be 36Mb/s.  4000*8/36 = 888seconds ~ 14.8 minutes.

Labs can override this.
Labs can increase the chapter to 12GB per chapter.
Labs can also increase the bitrate to 100Mb/s (I only recommend 90Mb/s for stable captures).   

Use ```*BITR=90*64BT=1``` to get the most quality and larger chapters.

**Reply by i-marinkovic** [reply]
Great, a very good explanation! I’m going to check ASAP.
